community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/helpers/JavaConversionSupport.scala

```diff
@@ -42,4 +41,33 @@ object JavaConversionSupport
     def hasNext = iterator.hasNext
     def next() = f(iterator.next())
   }
+
+  // Same as mapToScala, but handles concurrency exceptions by swallowing exceptions
+  def mapToScalaENFXSafe[T](iterator: PrimitiveLongIterator)(f: Long => T): Iterator[T] = new Iterator[T] {
```

Comment:
Sure, but it is an abstract class. We use it from scala as well, here for example
https://github.com/neo4j/neo4j/blob/2.3/community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/ExpandIntoPipe.scala#L98

But it is fine the way it is as well.